.github/dependabot.yml

```diff
@@ -61,26 +85,22 @@ updates:
       interval: weekly
       day: sunday
       time: '22:00'
-    open-pull-requests-limit: 0
+    open-pull-requests-limit: 5
     versioning-strategy: increase
     ignore:
-      # Only allow patch as minor babel versions need to be upgraded all together
-      - dependency-name: '@babel/*'
-        update-types:
-          - 'version-update:semver-major'
-          - 'version-update:semver-minor'
-
+      # only accept security updates
       - dependency-name: '*'
         update-types:
           - 'version-update:semver-patch'
+          - 'version-update:semver-minor'
```

Comment:
you are literally saying to ingore all types of upgrades here I think :)

Reply:
Ok in that case we should entirely remove snyk so we don't have 2 tools working on security dependencies. 

It's definitely unclear that this config will still allow security bumps

Reply:
I did some more research and it seems like this doesn't work anymore and dependabot v2 made "security updates only" only available through the github ui :(

So I will remove this ignore statement and put back the one for babel, and we'll just have to manage security vs non-security updates in the github settings. I personally prefer that for now, internally upgrading packages for non-security updates doesn't offer us anything; if we need a feature from a newer package we'll just upgrade it manually.

Reply:
And in that case, this PR doesn't appear to do anything except add some groupings; once security updates are enabled in the github UI, in theory they ignore the PR limit set in dependabot.yml